core/src/main/codegen/templates/Parser.jj

```diff
@@ -623,8 +630,7 @@ SqlNode OrderedQueryOrExpr(ExprContext exprContext) :
 {
     SqlNode e;
     SqlNodeList orderBy = null;
-    SqlNode start = null;
-    SqlNode count = null;
+    SqlNode[] startCount = {null, null};
 }
 {
```

Comment:
Is it necessary to use Array to indicate the first value is the offset and the second value is the limit? It maybe not a good practice.

Reply:
Hi @chunweilei , thanks for your review, I'm glad to hear your suggestions to improve it.